spec/models/card_spec.rb

```diff
@@ -1,33 +1,103 @@
 require 'rails_helper'
 
-describe Card do
-  let(:card) { Card.create(original_text: 'Привет', translated_text: 'ПриВЕТ') }
+  describe Card do
+    let!(:card) { build(:card) }
+    let!(:wrong_card) { Card.create(original_text: 'Привет',
+                                   translated_text: 'ПривЕТ') }
 
-  context '#words_cannot_be_equal' do
-    let(:card) { Card.new(original_text: 'Good', translated_text: 'GoOd') }
+    context '#words_cannot_be_equal' do
 
-    it 'does not pass the same words' do
-      expect(card).not_to be_valid
+      it 'does not pass the same words' do
+        expect(wrong_card).not_to be_valid
+      end
     end
-  end
 
-  context '#add_review_date' do
-    it 'equals time now + 3days' do
-      expect(card.review_date).to eq(Date.today + 3.days)
+    context '#prepare_word' do
+      it 'work with cyrillic' do
+        expect(wrong_card).not_to be_valid
+      end
     end
-  end
 
-  context '#prepare_word' do
-    it 'work with cyrillic' do
-      expect(card).not_to be_valid
+    context '#check_answer' do
+      context 'when answer is correct' do
+        let!(:correct_answer) { card.check_answer('Hello') }
+
+        it 'user\'s answer equals to original text' do
+          expect(correct_answer).to be true
+        end
+
+        it 'increment field correct_answers by 1' do
+          expect(card.correct_answers).to eq 1
+        end
+      end
+
+      context 'when answer is incorrect' do
+        let!(:incorrect_answer) { card.check_answer('Aloha') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.